implementation/src/main/java/io/smallrye/mutiny/operators/UniCache.java

```diff
@@ -48,12 +51,7 @@ protected void subscribing(UniSerializedSubscriber<? super I> subscriber) {
                     break;
                 case COMPLETED:
                     // Result already computed
-                    subscribers.add(subscriber);
```

Comment:
Where was this subscriber removed from the list of `subscribers`?
Am I right, that it stays right now and may be notified again after my changes?

Reply:
moved the add to only the "invalid cache; resubscription" branch, all tests continue running.